packages/terra-form/src/_Utilities.jsx

```diff
@@ -0,0 +1,37 @@
+const autoResizeTextareaHeight = (onChange, baseRowCount) =>
+  (e) => {
```

Comment:
I would rename e to spell it out what it is a bit more

Reply:
Spelled out here. https://github.com/cerner/terra-core/pull/966/commits/4a949d8e4efe5e688a7fc02f51f8662e4e96bae3